lib/Compile.js

```diff
@@ -9,15 +9,32 @@ var Compile = module.exports = function (options) {
       
       var width = options.width;
       var jsonData = options.data;
+      
+      if(options.stickerSheet) {
+        var startLength = jsonData.length-1;
+        
+        for(var i=1; i<139; i++) {
```

Comment:
I see this number 139 showing up in a few places.  Can we make it a shared constant or something?